Sources/ArcGISToolkit/Components/Offline/PreplannedListItemView.swift

```diff
@@ -17,10 +17,13 @@ import ArcGIS
 
 @MainActor
 @preconcurrency
-public struct PreplannedListItemView: View {
```

Comment:
Please remove the `public` modifier from all the other places in this view where it isn't needed.